gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/begin.ts

```diff
@@ -0,0 +1,71 @@
+import { Result } from '@chainlink/gauntlet-core'
```

Comment:
These files add back commands that were removed in #208, should remove them and update the new commands instead.